### PR TITLE
feat:add attributes to Organization, Resource and Network

### PR DESCRIPTION
**Pull request by asulis**
### Description:

This pull request updates the Organization, Resource and Network entities by adding: 

 - Description, contactEmail and uri attributes to Organization; 
 - contactEmail and uri attributes to Resource;
 - description attribute to Network 

### Checklist:

_Make sure you tick all the boxes below if they are true or do not apply before you ask for review_

- [x] I have performed a self-review of my code
- [x] I have made my code as simple as possible
- [x] I have added relevant tests for my changes and the code coverage has not dropped substantially
- [x] I have removed all commented code
- [x] I have updated the documentation in all relevant places (Javadoc, Swagger, MDs...)
- [x] I have described the PR and added a meaningful title in the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format

### CODE REVIEW

**Review by RadovanTomik — CHANGES_REQUESTED** [review]
looks good, please add unit tests for the entities and mappers and add URL to Organization and Resource

**Comment by asulis** [comment]
> looks good, please add unit tests for the entities and mappers and add URL to Organization and Resource

Fixed in [42ee1cf](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/42ee1cf8be2d756b86abc03389394e3d96adf09b), [921e910](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/921e91088e769e509950196ff9176f4fa2e1130a), [6df5943](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/6df5943417ed275e224f7473a87857a92227417d), [64c3c53](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/64c3c536862d1aa7f12642c3f0e5f196ea4a4106)

File: src/main/java/eu/bbmri_eric/negotiator/governance/organization/OrganizationServiceImpl.java
Comment (RadovanTomik) [inline src/main/java/eu/bbmri_eric/negotiator/governance/organization/OrganizationServiceImpl.java:1]: please use mappers

File: src/main/java/eu/bbmri_eric/negotiator/governance/network/Network.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 /**
  * Represents a Network entity in the database. A Network has a unique identifier, a URI, a name, a
- * contact email, a set of managers, and a set of resources.
+ * description, a contact email, a set of managers, and a set of resources.
```
Comment (RadovanTomik) [inline src/main/java/eu/bbmri_eric/negotiator/governance/network/Network.java:31]: I would remove the list of attributes from the JavaDoc

File: src/main/java/eu/bbmri_eric/negotiator/governance/resource/ResourceServiceImpl.java
Comment (RadovanTomik) [inline src/main/java/eu/bbmri_eric/negotiator/governance/resource/ResourceServiceImpl.java:1]: please use mappers

File: src/test/java/eu/bbmri_eric/negotiator/unit/mappers/ResourceModelMapperTest.java
Comment (RadovanTomik) [inline src/test/java/eu/bbmri_eric/negotiator/unit/mappers/ResourceModelMapperTest.java:1]: add similar tests for network and org mappers

**Review by RadovanTomik — CHANGES_REQUESTED** [review]

File: src/main/java/eu/bbmri_eric/negotiator/governance/resource/ResourceServiceImpl.java
Comment (asulis) [reply src/main/java/eu/bbmri_eric/negotiator/governance/resource/ResourceServiceImpl.java:1]: Fixed in [7a57acd](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/7a57acd7bd642327ca35da7bac8ddafaf8d48a26)

File: src/test/java/eu/bbmri_eric/negotiator/unit/mappers/ResourceModelMapperTest.java
Comment (asulis) [reply src/test/java/eu/bbmri_eric/negotiator/unit/mappers/ResourceModelMapperTest.java:1]: Fixed in [740752b](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/740752bdf4d91bffbf5ed8ae371749d661e8ed12)

File: src/main/java/eu/bbmri_eric/negotiator/governance/organization/OrganizationServiceImpl.java
Comment (asulis) [reply src/main/java/eu/bbmri_eric/negotiator/governance/organization/OrganizationServiceImpl.java:1]: Fixed in [7a57acd](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/7a57acd7bd642327ca35da7bac8ddafaf8d48a26)

File: src/main/java/eu/bbmri_eric/negotiator/governance/network/Network.java (hunk 1 above)
Comment (asulis) [reply src/main/java/eu/bbmri_eric/negotiator/governance/network/Network.java]: Fixed in [dd39ca8](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/dd39ca81c88f841010c840aba8962026ac73eb54)

**Review by RadovanTomik — APPROVED** [review]

**Pull request merged**